journeyman/webdev/svg.md

```diff
@@ -0,0 +1,25 @@
+# SVG (Scalable Vector Graphics)
+
+## Intro
+
+SVG is a Scalable Vector Graphics, which basically means it is a language used for to create two-dimensional vector graphics. These graphics can be used to create complex or simple graphics that can be animated in a variety of ways.
+
+## Suggested Learning
+
+- Give yourself a basic feel for SVG on [W3Schools.](https://www.w3schools.com/graphics/svg_intro.asp)
+- Learn other, more common ways to use SVG at [css-tricks.](https://css-tricks.com/using-svg/)
+- Learn how to use the `<path>` element [here.](https://developer.mozilla.org/en-US/docs/Web/SVG/Tutorial/Paths)
+
+## Requirements
+- Make a project that demonstrats the points below:
```

Comment:
Have them create an SVG with either a desktop tool like Sketch, Adobe Illustrator/Photoshop, etc or an online tool like http://editor.method.ac

All of those programs support exporting as SVG.